app/controllers/articles_controller.rb

```diff
@@ -2,38 +2,55 @@
 
 class ArticlesController < ApplicationController
   before_action :set_article, only: %i[show edit update destroy]
+  before_action :set_user, only: %i[show edit update destroy]
 
   def index
-    @articles = Article.all
+    @articles = policy_scope(Article)
```

Comment:
It is ok :ok_hand:,  but you can also use 
```suggestion
    @articles = ArticlePolicy::Scope.new(current_user, Article).resolve
```